packages/api/src/services/security/mappers/granted-authorities-ui-model.mapper.ts

```diff
@@ -0,0 +1,59 @@
+import {AuthorityList} from '../../../models/security/authority-list';
+import {Mapper} from '../../../providers/mapper/mapper';
+import {Authority} from '../../../models/security/authority';
+
+/**
+ * Mapper class for converting Authority arrays to AuthorityList models.
+ */
+export class GrantedAuthoritiesUiModelMapper extends Mapper<AuthorityList> {
+  /**
+   * Maps an array of Authority objects to a UI AuthorityList model.
+   *
+   * @param data - An array of Authority objects to be mapped into an AuthorityList.
+   * @returns A new AuthorityList instance containing the provided authorities.
+   */
+  mapToModel(data: string[]): AuthorityList {
```

Comment:
I get this with my scanner. can it be refactored?

<img width="908" height="224" alt="Image" src="https://github.com/user-attachments/assets/c2aca7f8-bebf-4660-8a3b-cf45c9004e91" />

Reply:
refactored